### PR TITLE
Create v0.1.1 release

### PR DESCRIPTION
**Pull request by LDSamson**
Includes bug fixes for issues #125 and #108

### CODE REVIEW

**Comment by LDSamson** [comment]
Hi both, let me know if I missed anything here for this bug fix release

File: vignettes/test_results.rds
Comment (jthompson-arcus) [inline vignettes/test_results.rds:1]: Is this manually updated on your machine?

**Comment by jthompson-arcus** [comment]
> Hi both, let me know if I missed anything here for this bug fix release

@LDSamson you are missing the `pkgdown` GHA workflow. It may as well be added in this PR.

File: vignettes/test_results.rds
Comment (LDSamson) [reply vignettes/test_results.rds:1]: Yes, that is correct. Created within the Docker image that we use for production. Would be great to automate that but that is on the to do list

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# clinsight 0.1.1
+
+## Changed 
+
+- Added `pkgdown` GHA workflow to automatically update documentation site with PRs & pushes to `main` and `dev`
```
Comment (jthompson-arcus) [inline NEWS.md:5]:
```suggestion
- Added `pkgdown` GHA workflow to automatically update documentation site with PRs & pushes to `main`
```

**Pull request merged**